frontend/src/lib/components/PropertiesTable.tsx

```diff
@@ -104,7 +104,7 @@ function ValueDisplay({ value, rootKey, onEdit, nestingLevel }: ValueDisplayType
     )
 
     return (
-        <div className="properties-table-value">
+        <div className="properties-table-value ph-no-capture">
```

Comment:
I think we can be more precise here and only sanitize the actual value as opposed to the type of value indicator, extra link, etc.